### PR TITLE
Ignore errors during tracing to not crash the cli

### PR DESCRIPTION
**Pull request by jugglingthebits**
I am not sure this is the right position within the codebase to handle syntax errors in TS files, but as far as I can see this fix works for #510 .

### CODE REVIEW

**Comment by JeroenVinke** [comment]
Thanks for creating the issue and the PR! This is one of those spots where we should add some logging as part of https://github.com/aurelia/cli/issues/504. So i'm not sure if swallowing the errors here is the best solution for this problem. We definitely want to prevent the watcher from crashing, but this may swallow other errors that we do want to show

**Comment by jugglingthebits** [comment]
I see. I thought that is a different issue as in my case it isn't swallowing the error at all - quite the opposite.

**Comment by JeroenVinke** [comment]
Yes, you're right, it is a different issue. But we want to keep #504 in mind here as we don't want to swallow errors that we don't have to

**Comment by AStoker** [comment]
Having been bit by swallowed errors before, I'm hesitant to just just straight up swallow without any kind of logging. What can happen is some bad code might be pulled down, and the compilation will fail, but not give the user any kind of indication on what just happened, or where. No bueno there 😞

File: lib/build/bundled-source.js [hunk 1]
```diff
@@ -122,6 +122,8 @@ exports.BundledSource = class {
           }, this.includedBy);
         }
       }
+    }, traceReject => {
+      // Ignore failed traces
```
Comment (EisenbergEffect) [inline lib/build/bundled-source.js]: Can we log the errors at least, instead of ignoring them?

**Comment by jugglingthebits** [comment]
I will just close this, as #510 has been fixed elsewhere.

**Pull request closed without merging**